cluster.go

```diff
@@ -18,6 +19,10 @@ var errClusterNoNodes = internal.RedisError("redis: cluster has no nodes")
 // ClusterOptions are used to configure a cluster client and should be
 // passed to NewClusterClient.
 type ClusterOptions struct {
+	// The network type, either tcp or unix.
+	// Default is tcp.
+	Network string
```

Comment:
I don't think unix sockets are used with Redis Cluster, because you need TCP/IP anyway for Redis Cluster to work.